.pre-commit-config.yaml

```diff
@@ -1,19 +1,24 @@
 ---
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v4.3.0
     hooks:
       - id: check-yaml
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/psf/black
-    rev: 22.3.0 # This should be kept in sync with the version in requirements-dev.in
+    rev: 22.10.0 # This should be kept in sync with the version in requirements-dev.in
     hooks:
       - id: black
         language_version: python3
         files: gnomad
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: v1.7.0 # This should be kept in sync with the version in requirements-dev.in
```

Comment:
Slightly different than the reqiurements-dev.in which omits the `v`  in `v1.7.0`, not sure if this is needed or if it can be removed here as well?

Reply:
I think it needs to match the tag in the github repo https://github.com/hhatto/autopep8/releases/tag/v1.7.0